src/Control/HTTPRequest.php

```diff
@@ -434,8 +434,7 @@ public function isAjax()
      * @param string $offset
      * @return bool
```

Comment:
```suggestion
```
There's no need to keep phpdocs that only suggest a type - this at best is redundant and at worst can introduce inconsistencies if one of these changes but the other doesn't.

This applies to other PHPDocs for arguments and return types that you have explicitly defined in this and related PRs.

Reply:
Done.